{"name": "Division", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\dell\\Desktop\\Bootcamp\\CICD\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\dell\\Desktop\\Bootcamp\\CICD\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\calcSteps.py\", line 23, in step_then_check_answer\n    assert actual_answer == expected_answer\n"}, "steps": [{"name": "Given I have a calculator", "status": "passed", "start": 1689138371929, "stop": 1689138371930}, {"name": "When I enter \"10\" as the first number", "status": "passed", "start": 1689138371930, "stop": 1689138371931}, {"name": "And I enter \"/\" as the operation", "status": "passed", "start": 1689138371932, "stop": 1689138371933}, {"name": "And I enter \"2\" as the second number", "status": "passed", "start": 1689138371933, "stop": 1689138371934}, {"name": "Then the answer should be \"5\"", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\dell\\Desktop\\Bootcamp\\CICD\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\dell\\Desktop\\Bootcamp\\CICD\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\calcSteps.py\", line 23, in step_then_check_answer\n    assert actual_answer == expected_answer\n"}, "start": 1689138371935, "stop": 1689138371938}], "start": 1689138371926, "stop": 1689138371939, "uuid": "ac72b2c7-a24d-4c04-97d1-d4c4d4094ab6", "historyId": "ba97aaca913a6129cf0224cf97d9d863", "fullName": "Calculator: Division", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Calculator"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}